magda-typescript-common/src/semantic-indexer/commonSemanticIndexerYargs.ts

```diff
@@ -0,0 +1,41 @@
+import yargs from "yargs";
+import { MinionArguments, commonYargs } from "@magda/minion-sdk";
```

Comment:
@magda/minion-sdk might not be built yet at this stage as it depends on minion-framework and minion-framework is built on typescript-common.
It might work on your local as you probably already build @magda/minion-sdk previously (i.e. built result is already in @magda/minion-sdk/dist/* )